apps/examples/testcase/ta_tc/device_management/utc/utc_dm_lwm2m.c

```diff
@@ -36,8 +36,8 @@
 
 extern dm_lwm2m_client_state_e dm_state;
 
-struct dm_lwm2m_context_s test_data1 = { {"192.168.1.152", "5683", false}, {300} };
-struct dm_lwm2m_context_s test_data2 = { {"192.168.1.152", "123AB", false}, {300} };
+struct dm_lwm2m_context_s test_data1 = { {CONFIG_UTC_DM_SERVER_IPADDR, CONFIG_UTC_DM_SERVER_PORT, false}, {300} };
```

Comment:
Here is the only place that using this addr & port, then adding in Kconfig is not neccessary, I think
Define is much easier for user, I think.
Choice is yours =)